godotproject/enemy-Ant.gd

```diff
@@ -42,6 +46,11 @@ func _physics_process(_delta):
 			if is_on_wall():
 				direction *= -1
 
+func take_damage(attack_damage):
+	health = health - attack_damage
+	if health <= 0:
+		dead()
+
```

Comment:
take out all the code that is a part of FGTS-81

Reply:
take damage

Reply:
removed